### ISSUE TITLE
webpack4 support?

### ISSUE THREAD

**Issue by phun-ky** [GProst/webpack-clean-obsolete-chunks#28]
When can this be in place? :)

I love this plugin, but we currently get this warning in the logs:

```
(node:3034) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
    at CleanObsoleteChunks.apply (/mnt/c/Users/v6a.EUROPE/Workspace/MyBusiness.Frontend/node_modules/webpack-clean-obsolete-chunks/index.js:19:12)
```

**Comment by jdelStrother** [comment]
You might give the latest release a try - `yarn add -D webpack-clean-obsolete-chunks@next`.

**Comment by jdelStrother** [comment]
This is now released as 0.4.0.
